.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
```

Comment:
well, the docs do not mention [report] here https://coverage.readthedocs.io/en/coverage-4.2/source.html#execution
anyway, I updated it as per your example, but is still looking for cms_form_example.
Better look at this tomorrow, instead of checking it in the middle of my baby's bath :smile: